src/one_dragon/base/operation/one_dragon_context.py

```
@@ -331,6 +330,15 @@ def init_async(self) -> None:
         f = ONE_DRAGON_CONTEXT_EXECUTOR.submit(self.init)
         f.add_done_callback(thread_utils.handle_future_result)
 
+    def start_btn_listener(self) -> None:
+        """
+        启动按键与鼠标监听器。
+        推迟到程序启动与 GUI 初始化完成之后显式调用，避免初始化阶段加载重型 C 扩展库
```

Ruff (0.16.0)

[warning] Docstring contains ambiguous `，` (FULLWIDTH COMMA). Did you mean `,` (COMMA)?

(RUF002)